### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by BryanNee** [1101-GS4719-C/Python-Community-GS4719-C#104] [Q&A]
想請問大家在使用jupyter lab 的input語法時，當想要更改輸入值時，jupyter lab 的kernal 會斷掉，需要重新連接到python3.9。想請問大家是否有這個問題?

**Comment by enoch-lightning** [comment]
你這樣講，我看不懂。如果你可以截圖，會比較容易讓人明白。你文中的kernal應該是kernel。

**Reply by enoch-lightning** [reply]
Python的kernel應該就是Python的「直譯器」，也可說是「解釋器。」就是從[Python官網：python.org](https://www.python.org/)中下載的python.exe。
<img width="960" alt="python直譯器" src="https://user-images.githubusercontent.com/86505454/146373565-13ec3196-ab45-4a81-a289-50b86962d45a.png">
